File: test-suite/src/test/java/io/micronaut/guice/doc/examples/bindings/annotations/Application.java
package io.micronaut.guice.doc.examples.bindings.annotations;

// tag::imports[]
import io.micronaut.guice.annotation.Guice;
import io.micronaut.runtime.Micronaut;
// end::imports[]

// tag::class[]
@Guice(modules = CreditCardProcessorModule.class) // <1>
public class Application {

    public static void main(String[] args) {
        Micronaut.run(Application.class, args);
    }
}
// end::class[]

File: test-suite-kotlin/src/test/kotlin/io/micronaut/guice/doc/examples/bindings/annotations/Application.kt
package io.micronaut.guice.doc.examples.bindings.annotations

// tag::imports[]
import io.micronaut.guice.annotation.Guice
import io.micronaut.runtime.Micronaut
// end::imports[]

// tag::class[]
@Guice(modules = [CreditCardProcessorModule::class]) // <1>
class Application {

    companion object {
        @JvmStatic
        fun main(args: Array<String>) {
            Micronaut.run(Application::class.java, *args)
        }
    }
}
// end::class[]
